resources/resource-extensbility/2024-06-resource-extensibility-feature-spec.md

```diff
@@ -0,0 +1,432 @@
+    -  They want to use the project at their work and need a feature, 
+    -  The project triggered their interests in the cloud native landscape and want to use the technology.
+    -  They want to code in a particular language E.g. :Go
+    -  They want to integrate their existing tools and technologies with the project
+
+Today Radius enables users to get started on contributing to Radius with good-first-issues but doesn’t have a model to promote contributions further. Since the beginning of its open source launch, Dapr created the components repository and invited users to build their own components to unblock their scenarios and in turn folks contributed to the project. Learning from Dapr, Radius needs to have the extensibility points defined for contributors to interact with the project so that they can build out the things that they need to enable their scenarios and in turn the community gets benefitted with those contributions.
+
+### Customer persona(s)
+<!-- Who is the target customer? Include size/org-structure/decision makers where applicable. -->
+- Platform engineers: Platform engineers are responsible for streamlining the developer-experience for their organization by defining a set of recommended practices for application teams and provide self-service capabilities for application teams/developers to use. They are responsible for building the platform and providing the necessary tools and services for the application teams to deploy their applications.
+
+- IT Operators: IT operators are responsible for managing the infrastructure and ensuring that the applications are running smoothly. They are responsible for maintaining the infrastructure and providing support for the infrastructure. They are the primary users of Radius Recipes as they are responsible for defining the infrastructure-as-code templates for the applications.
+
+- System Reliability Engineers (SREs) : SREs are responsible for ensuring the reliability of the applications and services that are deployed in the cloud. They are responsible for maintaining the infrastructure and ensuring that the applications are running smoothly. They maintain the infrastructure-as-code templates and provide support for the applications that are deployed. 
+
+- System integrators : System integrators are partners who help enterprises integrate proprietary services with cloud native applications. They are responsible for designing and building cloud native applications on behalf of their customers.
+
+- Developers : Developers are responsible for building the cloud native applications. They are responsible for writing the code, designing and maintaining the applications.
+
+- Open-source contributors : Cloud native open-source contributors can be any of the above personas who are interested in contributing to the cloud native projects. 
```

Comment:
❤️ - I love that we're pointing out the OSS aspects for all of the above personas.